deploy/charts/kubernetes-cluster-prep/templates/NOTES.txt

```diff
@@ -0,0 +1,31 @@
+The Conjur/Authenticator Namespace preparation is complete.
+The following have been deployed:
+{{ if .Values.authnK8s.configMap.create }}
+A Golden ConfigMap
+{{ end }}
+{{ if .Values.authnK8s.rbac.create }}
+An authenticator ClusterRole
+{{ end }}
+{{ if .Values.authnK8s.serviceAccount.create }}
+An authenticator ServiceAccount
+{{ end }}
+
+{{ if .Values.displaySamplePolicy }}
```

Comment:
Please delete everything from 13 to the end of file. This will get added with the `helm test` commit.